mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+ return price + price*0.2;
```

Comment:
Minor things to suggest. Please keep spacing consistent.You did it for +, but * needs space.
I had the same problem like you for consider spacing.